packages/react/package.json

```diff
@@ -57,6 +57,7 @@
     "@vanilla-extract/recipes": "^0.3.0",
     "@vanilla-extract/sprinkles": "^1.5.1",
     "compress.js": "^1.2.2",
+    "country-flag-emoji-polyfill": "^0.1.4",
```

Comment:
@ahmed-gz this is no longer used, correct?

Reply:
@ikovic it's still used actually to show the flag for the selected country in the form